sockeye/inference.py

```diff
@@ -1161,16 +1197,19 @@ def _concat_translations(self, translations: List[Translation]) -> Translation:
 
     def _translate_nd(self,
                       source: mx.nd.NDArray,
-                      source_length: int) -> List[Translation]:
+                      source_length: int,
+                      raw_constraints: List[Optional[constrained.RawConstraintList]]) -> List[Translation]:
         """
         Translates source of source_length, given a bucket_key.
 
         :param source: Source ids. Shape: (batch_size, bucket_key, num_factors).
         :param source_length: Bucket key.
+        :param constraint_list: A list of constraint sets (one or more constraints per sentence).
```

Comment:
param name is outdated.
Also I would write something like "A list of optional constraint sets".